Readme.md

```diff
@@ -8,7 +8,7 @@
 [![Tests](https://github.com/i-love-flamingo/flamingo/workflows/Tests/badge.svg?branch=master)](https://github.com/i-love-flamingo/flamingo/actions?query=branch%3Amaster+workflow%3ATests)
 [![Release](https://img.shields.io/github/release/i-love-flamingo/flamingo?style=flat-square)](https://github.com/i-love-flamingo/flamingo/releases)
 [![TODOs](https://badgen.net/https/api.tickgit.com/badgen/github.com/i-love-flamingo/flamingo)](https://www.tickgit.com/browse?repo=github.com/i-love-flamingo/flamingo)
-[![Join the chat at https://discord.gg/hmHgm3pYNn](https://img.shields.io/badge/Discord-%235865F2.svg?style=plasticsquare&logo=appveyor&label=join&nbsp;us&nbsp;at)](https://discord.gg/hmHgm3pYNn)
+[![Slack](https://img.shields.io/badge/Gophers_slack-%23flamingo-white?style=social&logo=slack&logoColor=E72064)](https://gophers.slack.com/archives/C04QYKWLVPD)
```

Comment:
provided link does not work.
```suggestion
[![Slack](https://img.shields.io/badge/Gophers_slack-%23flamingo-white?style=social&logo=slack&logoColor=E72064)](https://gophers.slack.com/messages/flamingo)
```

Reply:
This link opens the slack app directly. while the messages link goes to the browser view